.github/workflows/package-test.yml

```diff
@@ -1,12 +1,12 @@
 # HMS Networks Solution Center
-# Compile Test Action for Maven-based Ewon ETK Projects
+# Package Test Action for Maven-based Ewon ETK Projects
 # Version: 2.1
 #
 # This action is configured to automatically run when a push
 # is made or pull request is merged to the `main` or `master`
 # branch.
 
-name: Compile Test
+name: Package Test
```

Comment:
What do you think about `Package Build Test` or `Packaging Test`? I don't think it's a huge concern, but `Package Test` just seems off to me.

Reply:
This is still in draft. not sure why i added a reviewer yet.

Im fine with "Package Test" or "Build Test", "Package Test" doesnt bother me at all.

Reply:
I'm okay with either, although `Build Test` may be a bit more descriptive as it covers the Javadoc checking too.